frontend/src/components/TwoLevelDropdown.tsx

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright 2022 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Ref, useRef, useState } from 'react';
+import { Button, Tooltip } from '@material-ui/core';
+import { color } from 'src/Css';
+import { classes, stylesheet } from 'typestyle';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+
+const css = stylesheet({
```

Comment:
I'm happy to make that change - what's the benefit of that extraction? I only kept it within the same file since that seemed to be the norm amongst the frontend components ([[1]](https://github.com/kubeflow/pipelines/blob/c5df3e88cc48580e7c4ffba03a163e9fddbaf3fd/frontend/src/components/Graph.tsx#L41), [[2]](https://github.com/kubeflow/pipelines/blob/c5df3e88cc48580e7c4ffba03a163e9fddbaf3fd/frontend/src/components/Banner.tsx#L32), [[3]](https://github.com/kubeflow/pipelines/blob/c5df3e88cc48580e7c4ffba03a163e9fddbaf3fd/frontend/src/components/Metric.tsx#L24)).

Reply:
If you think these stylesheet are specific to this file, it is okay to keep them here. However, if this is a common stylesheet that should be kept consistent across the app, you can move them to https://github.com/kubeflow/pipelines/blob/c5df3e88cc48580e7c4ffba03a163e9fddbaf3fd/frontend/src/Css.tsx#L201